src/cli/index.mjs

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env node --experimental-modules
+const map = (fn, arr) => !arr ? cArr => cArr.map(fn) : arr.map(fn)
+
+console.log(map(a => a * 2, [2, 3, 4]))
```

Comment:
Why use console.log?

Reply:
This is the CLI script for FusionBook. Essentially, the purpose of console.log here is to print some information out to stdout.
The CLI tool of the user takes care of displaying this information. In order to ascertain that the CLI was indeed functional, some output needed to be given to the terminal. Hence, the console.log.

Reply:
Cool